helm/aws-load-balancer-controller/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: aws-load-balancer-controller
 description: AWS Load Balancer Controller Helm chart for Kubernetes
-version: 1.7.1
-appVersion: v2.7.0
+version: 1.7.3
```

Comment:
Based on https://github.com/aws/eks-charts/blob/master/stable/aws-load-balancer-controller/Chart.yaml#L5 `1.7.2` was already released

Reply:
please revert the change in the version numbers, we will update the version during our helm chart release, thanks.